### DISCUSSION TITLE
Show the date in the tag list

### DISCUSSION THREAD

**Discussion by Clasyc** [klausmeyer/docker-registry-browser#595] [Ideas]
It would be great to display the date and time when a tag was created/updated in the tags list.

(it could be located either on 1 or 2 mark)
![image](https://github.com/klausmeyer/docker-registry-browser/assets/17416365/428700cc-5d06-4666-baff-7b5ecd35911b)

BTW, really great project, thanks a lot. :100:

**Comment by klausmeyer** [comment]
> BTW, really great project, thanks a lot. 💯

Thanks for this very nice feedback 🙇 I like your idea but unfortunately the things I can implement in this application are limited by the API provided by the docker registries.

We have some feature requests as issues also addressing very similar things:

- https://github.com/klausmeyer/docker-registry-browser/issues/291
- https://github.com/klausmeyer/docker-registry-browser/issues/520

At the moment I don't see a way how to archive implementing features like this without a complete change in how the application works - which is not something I plan to do.

Background:

The docker registry API does not include information about the tags in the [endpoint](https://distribution.github.io/distribution/spec/api/#listing-image-tags) listing them. It just returns an array of tag names like `["latest", "v2", "v1", "nightly"]`. So I would need to send another API request **per tag** to fetch the actual details of the tag which would be resulting in a classic [N+1 problem](https://restfulapi.net/rest-api-n-1-problem/).

One of the key decisions when building this application was to build it in a stateless way. There is no need for a database or any background processes which synchronise data with the connected registries - instead each request to the application triggers one request to the registry to fetch the data directly. This keeps the deployment of the application also very simple.
